make.config.in

```diff
@@ -58,7 +58,9 @@ FACETS_SOURCE   = @FACETS_SOURCE@
 # Files that are needed by configure and should be dependencies for 'all'
 
 OBJ						= $(SOURCEC:%.cxx=%.o)
+ifndef RELEASE
```

Comment:
I don't understand what's going on here. Why only compile libbout if `RELEASE` is set? What are the other options (`DEBUG`?), and what gets compiled for them?